main-project/src/chats/chats-controller.service.ts

```diff
@@ -30,35 +29,17 @@ import { ChatUsersRepository } from './repository/chat-users.repository';
 @Injectable()
 export class ChatsControllerService {
   constructor(
-    @InjectRepository(ChatUsersRepository)
     private readonly chatUsersRepository: ChatUsersRepository,
-
-    @InjectRepository(ChatLogRepository)
     private readonly chatLogRepository: ChatLogRepository,
-
-    @InjectRepository(ChatListRepository)
     private readonly chatListRepository: ChatListRepository,
-
-    @InjectRepository(NoticesRepository)
-    private readonly noticesRepository: NoticesRepository,
-
-    @InjectRepository(NoticeChatsRepository)
     private readonly noticeChatsRepository: NoticeChatsRepository,
   ) {}
 
-  async getChatRoomsByUserNo(userNo: number): Promise<ChatRoom[]> {
-    const chatRooms: ChatRoom[] =
-      await this.chatUsersRepository.getChatRoomsByUserNo(userNo);
-
-    return chatRooms;
-  }
-
   async getPreviousChatLog(
-    getChatLogDto: GetChatLogDTO,
+    userNo: number,
     chatRoomNo: number,
+    { currentChatLogNo }: GetChatLogDTO,
```

Comment:
```suggestion
    { currentChatLogNo }: GetChatLogDto,
```
저희 Dto는 앞에꺼만 대문자쓰고있어요